docs/source/authentication.rst

```diff
@@ -98,25 +98,31 @@ LDAP
 The LDAP backend authenticates user against an LDAP server. The following is a list of
 configuration options for the backend.
 
-+----------+----------+---------+------------------------------------------------------------+
-| option   | required | default | description                                                |
-+==========+==========+=========+============================================================+
-| users_ou | yes      |         | OU of the user accounts                                    |
-+----------+----------+---------+------------------------------------------------------------+
-| host     | yes      |         | Hostname of the LDAP server                                |
-+----------+----------+---------+------------------------------------------------------------+
-| port     | yes      |         | Port of the LDAP server                                    |
-+----------+----------+---------+------------------------------------------------------------+
-| use_ssl  | no       | false   | Use LDAPS to connect                                       |
-+----------+----------+---------+------------------------------------------------------------+
-| use_tls  | no       | false   | Start TLS on LDAP to connect                               |
-+----------+----------+---------+------------------------------------------------------------+
-| cacert   | no       | None    | Path to the CA cert used to validate certificate           |
-+----------+----------+---------+------------------------------------------------------------+
-| id_attr  | no       | uid     | Field name of the user ID attribute                        |
-+----------+----------+---------+------------------------------------------------------------+
-| scope    | no       | subtree | Search scope (base, onelevel, or subtree)                  |
-+----------+----------+---------+------------------------------------------------------------+
++---------------+----------+---------+------------------------------------------------------------+
```

Comment:
@Kami Good idea, i'll be reading from the README in the enterprise auth backend repo though.  I'll come back to this. I don't want to tackle this today.